core/src/view/Sound.java

```diff
@@ -34,13 +34,13 @@ public void stopGameMusic(){
      * Plays the sound associated with the enemy hitting something/someone.
      */
     public void playEnemyHit(){
-        ENEMY_HIT.play(); // TODO: check if the sounds isnt too loud later
+        ENEMY_HIT.play(); // TODO: check if the sounds isn't too loud later
```

Comment:
to reduce the volume put a float inside () between 1 and 0. where 1 is full volume